docs/api/datetime.md

```diff
@@ -76,43 +67,48 @@ January always has a leading zero, such as `01`. Additionally, the hour is
 
 For example, if we wanted users to only select minutes in increments of 15, we could pass `"0,15,30,45"` to the `minuteValues` property.
 
 As another example, if we wanted users to only select from the month of October, we could pass `"10"` to the `monthValues` property.
 
-## Customizing Date and Time Presentation
-
-Some use cases may call for only date selection or only time selection. The `presentation` property allows you to specify which pickers to show and the order to show them in. For example, `presentation="time"` would only show the time picker. `presentation="time-date"` would show the time picker first and the date picker second, but `presentation="date-time"` would show the date picker first and the time picker second.
+TODO
 
-## Reset and Cancel Buttons
+### Advanced Date Constraints
 
-`ion-datetime` provides `cancel` and `reset` methods that you can call when clicking on custom buttons that you have provided in the `buttons` slot. The `reset` method also allows you to provide a date to reset the datetime to.
+With the `isDateEnabled` property, developers can customize the `ion-datetime` to disable a specific day, range of dates, weekends or any custom rule using an ISO 8601 date string. 
+The `isDateEnabled` property accepts a function returning a boolean, indicating if a date is enabled. The function is called for each rendered calendar day, for the previous, current and next month. Custom implementations should be optimized for performance to avoid jank.
 
-## Confirming Selected Values
+We recommend using a date utility such as `date-fns` when working with dates.
 
-By default, `ionChange` is emitted with the new datetime value whenever a new date is selected. To require user confirmation before emitting `ionChange`, you can either set the `showDefaultButtons` property to `true` or use the `buttons` slot to pass in a custom confirmation button. When passing in custom buttons, the confirm button must call the `confirm` method on `ion-datetime` for `ionChange` to be emitted.
+TODO
 
 ## Localization
 
 Ionic Framework makes use of the [Intl.DatetimeFormat](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DatetimeFormat) Web API which allows us to automatically localize the month and day names according to the language and region set on the user's device.
 
-For instances where you need a specific locale, you can use the `locale` property to set it. The following example sets the language to "French" and the region to "France":
+### Custom Locale
 
-```html
-<ion-datetime locale="fr-FR"></ion-datetime>
-```
+For instances where you need a specific locale, you can use the `locale` property to set it.
+
+TODO
 
-### Controlling the Hour Cycle
+### Hour Cycle
```

Comment:
Not related to these changes, but I just noticed the `h12` row in this section is missing a second `'`.